### DISCUSSION TITLE
Rename Remembered Databases

### DISCUSSION THREAD

**Discussion by trymeouteh** [keepassxreboot/keepassxc#10178] [Ideas]
Please add a feature to rename remembered databases within keepassxc. This will not rename the file but rename the listing in keepassxc.

For example. I have a remembered databased listed as "/home/john/Documents/Passwords.kdbx". I can rename it to "John's Passwords" and it will be listed as "John's Password". No more file path, no more extension type being in the name.

**Comment by o-alquimista** [comment]
I think this is already possible. Take a look at the "Database settings", in the "General" (first) tab.

![Screenshot of database settings](https://github.com/user-attachments/assets/1fd8e18f-7815-4d1e-af73-ac58abb19dce)

The setting I'm using ("Database name" in the "Database metadata" section) is only visible while the database is unlocked. If you want the display name to always be there, set it in the "Public database metadata" section below (this is stored unencrypted).
